ci-operator/step-registry/release-qe-tests/image-consistency-check/release-qe-tests-image-consistency-check-commands.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "Running Image Consistency Check"
+
+echo "Payload URL: $PAYLOAD_URL"
+echo "Merge Request ID: $MERGE_REQUEST_ID"
+
+gitlab_token=$(cat "/var/run/vault/release-tests-token/gitlab_token")
+export GITLAB_TOKEN=$gitlab_token
+
+# TODO oarctl image-consistency-check
```

Comment:
For now, this is not enabled. We need to firstly check if we can run the job with the environmental variables.